app/build.gradle

```diff
@@ -18,11 +18,16 @@ android {
     }
 }
 
+repositories {
+    google()
+}
+
 dependencies {
+    compile 'com.android.support:design:27.1.1'
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation 'com.android.support:appcompat-v7:27.1.1'
     implementation 'com.android.support.constraint:constraint-layout:1.1.2'
-    implementation 'com.android.support:design:27.1.1'
+
```

Comment:
And try to avoid empty lines in PRs as we want PR as clean as possible.... tomorrow I will talk about cleanliness on your PRs